BAZProject/MainMovie/MovieAPI.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MovieAPI.swift
+//  BAZProject
+//
+//
+
+import Foundation
+
+class MovieAPI {
```

Comment:
Considera marcar como final todas las clases para el próximo PR
```suggestion
final class MovieAPI {
```